examples/cellar/gen/http/storage/server/types.go

```diff
@@ -293,7 +293,7 @@ func NewMultiUpdatePayload(body *MultiUpdateRequestBody, ids []string) *storage.
 	return v
 }
 
-// Validate runs the validations defined on AddRequestBody
+// Validate runs the validations defined on addRequestBody
```

Comment:
That doesn't look right.... Is that solved in an upcoming PR? or maybe examples need to be regenerated?
(skipping other occurrences of this problem)